xs/src/libslic3r/PerimeterGenerator.cpp

```diff
@@ -6,6 +6,23 @@
 
 namespace Slic3r {
 
+int
+PerimeterGenerator::num_loops(int perimeters, int extra_perimeters)
+{
+    int generated_perimeters = perimeters + extra_perimeters - 1;
+    long min_shell_thickness = scale_(this->config->min_shell_thickness);
```

Comment:
@alexrj what is the context  for `coord_t` ? What is the assumption/guarantee with using it?